opqueue.go

```diff
@@ -126,19 +123,17 @@ func (q *OpQueue) Enqueue(id ID, op *Op) error {
 // If the OpQueue is closed, then Dequeue will return false
 // for the second parameter.
 func (q *OpQueue) Dequeue() (*OpSet, bool) {
-	q.cond.L.Lock()
```

Comment:
This is a pedantic change that frankly makes it a little less clear that one is using the same lock as the conditional.

Reply:
These changes should be a separate PR, adding cognitive load to a review that is core to how we process data isn't a smart idea.   We need to keep these kinds of PRs as simple as possible.